src/generators/minecraft-item/MinecraftItemGenerator.res

```diff
@@ -23,13 +23,68 @@ The generator supports four standard sizes:
 
 let images: array<Generator.imageDef> = [
   {id: "Background", url: Generator.requireImage("./images/Background.png")},
+  {id: "Title", url: Generator.requireImage("./images/Title.png")},
 ]
 
-let textures: array<Generator.textureDef> = TextureVersions.allTextureDefs
+let textures: array<Generator.textureDef> = Js.Array.concat(
+  TextureVersions.allTextureDefs,
+  [
+    {
+      id: "CenterFold",
+      url: Generator.requireImage("./textures/CenterFold.png"),
+      standardWidth: 2,
+      standardHeight: 128,
+    },
+  ],
+)
+let cycleTextureOffset = (t, tileWidth) => {
+  let t = if t === tileWidth {
+    0
+  } else {
+    t + 1
+  }
+  Belt.Int.toString(t)
+}
 
-let drawItem = (id, rectangle, x, y, size) => {
-  Generator.drawTexture(id, rectangle, (x, y, size, size), ())
-  Generator.drawTexture(id, rectangle, (x + size, y, size, size), ~flip=#Horizontal, ())
+let makeRegionId = (textureId, x, y, page) => {
```

Comment:
I'm thinking we make the `regionId` based on `textureId` and `rectangle` (the rectangle where it's positioned in the texture).

e.g.

```res
let makeRegionId = (textureId, rectangle) => {
  let (tileX, tileY, _, _) = rectangle
  textureId ++ "-" ++ Js.Int.toString(tileX) ++ "-" ++ Js.Int.toString(tileY)
}
```

This is so if you fix one of the textures, then it will fix all instances of that one texture, independent of scaling choice, position on page, and page it's on.

So if you create a page of just arrows for example, you only need to fix one, rather than having to fix each one of them individually.